share/exchange.yml

```diff
@@ -42,14 +42,17 @@ ASX_OTC:
       daily_close: 20h
       daily_open: -1h
       daily_settlement: 22h
+      trading_breaks:
+      - - 4h30m
```

Comment:
is this correct two `- -`

Reply:
https://github.com/yngshan/perl-Finance-Exchange/blob/6c676fa786585685fbb24d9dd272613d75026eb0/share/exchange.yml#L523

The same format applied to HKSE (Hong kong stock exchange)